cdap-ui/app/cdap/components/PipelineSummary/RunsHistoryGraph/index.js

```diff
@@ -25,11 +25,15 @@ import T from 'i18n-react';
 require('./RunsHistoryGraph.scss');
 require('react-vis/dist/styles/plot.scss');
 
+const MARKSERIESSTROKECOLOR = '#999999';
 const FAILEDRUNCOLOR = '#A40403';
 const SUCCESSRUNCOLOR = '#3cc801';
 const LINECOLOR = '#DBDBDB';
 const PREFIX = `features.PipelineSummary.runsHistoryGraph`;
 const GRAPHPREFIX = `features.PipelineSummary.graphs`;
+const ONE_MIN_SECONDS = 60;
+const ONE_HOUR_SECONDS = ONE_MIN_SECONDS * ONE_MIN_SECONDS;
```

Comment:
Maybe change this to `ONE_MIN_SECONDS * 60`, or add a constant for `ONE_HOUR_MIN`, since it took me a sec to get why ONE_HOUR_SECONDS = ONE_MIN_SECONDS ^ 2